docker/scripts/checkCustomFiles.sh

```diff
@@ -0,0 +1,6 @@
+if [[ -e /custom_files/ ]]; then
+	source /root/.bash_profile; \
+	sleep 5; \
+	/bin/cp --force /custom_files/cbiologo.png $CATALINA_HOME/webapps/cbioportal/images;
```

Comment:
In addition, I think you meant `/cbioportal/docker/customization/cbiologo.png`, although the default you suggest there is specific branding for Weil-Cornel, which is probably not the intended default to merge to rc.